media/css/firefox/nightly-firstrun.scss

```diff
@@ -20,7 +20,8 @@ main {
     padding-bottom: 0;
 
     .mzp-c-hero-title {
-        @include at2x('#{$image-path}/logos/firefox/browser/nightly/logo-lg.png', 124px, 124px);
+        background-image: url('#{$image-path}/logos/firefox/browser/nightly/logo-lg-high-res.png');
+        background-size: 124px 124px;
```

Comment:
I'm so sorry, I have once again led you astray... it looks like the background size should actually be 80px by 80px, which is inherited from the design system so we can just delete this line as well as the padding below.

```suggestion
```

It's not your fault, the 124px size that was there originally was already wrong and I hadn't tested on small screens. This bug is currently live but only appears in small windows, though this page gets very little mobile traffic. But we can fix it now by removing the size and padding and letting the standard styling take effect.

![image](https://user-images.githubusercontent.com/205591/90281808-191ff300-de22-11ea-8fcd-9d9ead9389fe.png)